HandyImage.user.js

```diff
@@ -2432,6 +2437,10 @@ function makeworld()
 	case "imageab.com":	
 		i = q('img[onclick*="(this"]');
 		break;
+	case "hizliresim.com":
+	case "hizliresimyukle.com":
+	case "yukleresim.com":
+	case "imgim.com":
```

Comment:
why did you put all the cases here before default: end case? It's pretty same as not putting cases at all

biggest image would be used as target image once page is finished loading

so websites aren't really added

if you need any help - just write example image links for above sites and i'll take a look